app/src/main/java/com/group80/uoftinder/ContactsActivity.java

```diff
@@ -0,0 +1,134 @@
+package com.group80.uoftinder;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.content.res.AppCompatResources;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.firebase.ui.firestore.FirestoreRecyclerAdapter;
+import com.firebase.ui.firestore.FirestoreRecyclerOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.Query;
+import com.group80.uoftinder.chat.ContactModel;
+import com.group80.uoftinder.firebase.storage.ImageStorageDbFacade;
+import com.group80.uoftinder.firebase.storage.StorageDbDownloadable;
+
+/**
+ * A window that displays all the contacts to chat with
+ */
+public class ContactsActivity extends AppCompatActivity {
```

Comment:
If you have the time, some side comments/documentation on what each method does would be great!